Makefile.dep

```diff
@@ -398,6 +398,10 @@ ifneq (,$(filter stdio_rtt,$(USEMODULE)))
 endif
 
 ifneq (,$(filter shell,$(USEMODULE)))
+  USEMODULE += stdio_rx
```

Comment:
could we maybe rename this module to "stdio_stdin" or just "stdin"? rx only makes sense for uart.

Reply:
+1
I saw your comment about that and I I agree with it.